wavefront_cli/lib/proxy.py

```diff
@@ -114,7 +114,7 @@ def configure_proxy(url, wavefront_api_token, csp_api_token=None,
     if csp_api_token:
         # replace csp api token
         cmd = ('sed', '-i', '-e',
-               f'/#cspAPIToken=/c\tcspAPIToken={csp_api_token})',
+               f'/#cspAPIToken=/c\tcspAPIToken={csp_api_token}',
```

Comment:
```suggestion
               f'/#cspAPIToken=/c\tcspAPIToken={csp_api_token} ',
```

Reply:
We'd also need a space

Reply:
for what ??

Reply:
Thought it was followed by a continuation string, but I see there's a comma, so we are fine